### ISSUE TITLE
Service beacon emitatstartup option broken

### ISSUE THREAD

**Issue by jagguli** [saltstack/salt#36876]
### Description of Issue/Question

Service beacon emitatstartup option broken

```
Oct 10 11:38:04 hawk2 salt-minion[2442]: [CRITICAL] The beacon errored:
Oct 10 11:38:04 hawk2 salt-minion[2442]: Traceback (most recent call last):
Oct 10 11:38:04 hawk2 salt-minion[2442]: File "/usr/lib/python2.7/dist-packages/salt/minion.py", line 2042, in handle_beacons
Oct 10 11:38:04 hawk2 salt-minion[2442]: beacons = self.process_beacons(self.functions)
Oct 10 11:38:04 hawk2 salt-minion[2442]: File "/usr/lib/python2.7/dist-packages/salt/minion.py", line 380, in process_beacons
Oct 10 11:38:04 hawk2 salt-minion[2442]: return self.beacons.process(b_conf)  # pylint: disable=no-member
Oct 10 11:38:04 hawk2 salt-minion[2442]: File "/usr/lib/python2.7/dist-packages/salt/beacons/__init__.py", line 92, in process
Oct 10 11:38:04 hawk2 salt-minion[2442]: raw = self.beacons[fun_str](b_config[mod])
Oct 10 11:38:04 hawk2 salt-minion[2442]: File "/usr/lib/python2.7/dist-packages/salt/beacons/service.py", line 106, in beacon
Oct 10 11:38:04 hawk2 salt-minion[2442]: if not service['emitatstartup']:
Oct 10 11:38:04 hawk2 salt-minion[2442]: TypeError: string indices must be integers, not str

```
### Setup

(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)
### Steps to Reproduce Issue

(Include debug logs if possible and relevant.)
### Versions Report

(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
Salt Version:
           Salt: 2016.3.3

Dependency Versions:
           cffi: 1.8.3
       cherrypy: 8.1.2
       dateutil: 2.5.3
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: 1.3.8
         Jinja2: 2.7.3
        libgit2: Not Installed
        libnacl: 1.5.0
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: 1.2.3
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.9 (default, Jun 29 2016, 13:08:31)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.4.0
           RAET: 0.6.3
          smmap: 0.8.2
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.5

System Versions:
           dist: debian 8.6
        machine: x86_64
        release: 3.16.0-4-amd64
         system: Linux
        version: debian 8.6

**Comment by jagguli** [comment]
Seems to be fixed in develop

**Comment by gtmanfred** [comment]
It looks like this bug was fixed here

https://github.com/saltstack/salt/pull/35199

I have tagged it to be backported to the 2016.3 branch

**Comment by rallytime** [comment]
@jagguli I have back-ported #35199 to the 2016.3 branch in #36945. Can you confirm that this issue is fixed for you on 2016.3?

**Comment by rallytime** [comment]
I'm closing this since the fix has been back-ported and merged. This will be available in the 2016.3.4 release.
